### DISCUSSION TITLE
getting bookmark urls from pinboard not working

### DISCUSSION THREAD

**Discussion by rainingkittens** [nianeyna/ao3downloader#151] [General]
Keep getting this message: 

`HTTPSConnectionPool(host='api.pinboard.in', port=443): Read timed out. (read timeout=30)`

Not sure what to do? Please help!

Thanks 😊

**Comment by nianeyna** [comment]
It appears that for some reason the pinboard api endpoint is timing out for you. I can only theorize why that would be, as it's not something I have any direct control over. The first thing I would do is try running the script using a different device and/or a different internet connection. If you can't do that or it doesn't work, try setting a cutoff date after which you only have a small number of bookmarks saved. If the cutoff thing works but nothing else does let me know so we can figure out a way to get the rest of your bookmarks too.

**Reply by rainingkittens** [reply]
Hi!

I turned on my vpn and that solved the issue, so I guess it had something to do with my internet? not sure why though 😅
Thanks for your suggestion, it is now working as it should!

Thanks again ❤️
